packages/website/src/services/emails.js

```diff
@@ -0,0 +1,8 @@
+const hostname = typeof window !== "undefined" ? window.location.hostname : "";
+const domain = hostname.substring(hostname.lastIndexOf(".", hostname.lastIndexOf(".") - 1) + 1);
+const emails = {
+  contact: `hello@${domain}`, // general contact email
+  report: `report@${domain}`, // report abuse email
```

Comment:
We are making a pretty heavy assumption here that the portal operator will have a certain set of emails configured for their domain. This might or might not be true. While certainly an improvement over the previously hardcoded email address, we can do better. Let's introduce an option for the portal op to customise these emails completely.

Reply:
redesigned to use WEBSITE_CONTACT_EMAIL and WEBSITE_ABUSE_EMAIL but default to hello@PORTAL_DOMAIN and report@PORTAL_DOMAIN respectively